frontend/src/hooks/api/secretApprovalRequest/queries.tsx

```diff
@@ -45,6 +45,78 @@ export const secretApprovalRequestKeys = {
   ]
 };
 
+export const decryptSecrets = (
+  encryptedSecrets: EncryptedSecret[],
+  decryptFileKey: UserWsKeyPair
+) => {
+  const PRIVATE_KEY = localStorage.getItem("PRIVATE_KEY") as string;
```

Comment:
Why is this still from local storage?

Reply:
Because it still used in places. Progressively removing it @maidul98

Reply:
oh okay cool, but before we we'll have it removed right?